dockerfiles/cli/version/5.0.0-M1/images

```diff
@@ -1,6 +1,6 @@
-IMAGE_SOCAT=codenvy/socat
+IMAGE_SOCAT=codenvy/socat:1.7.3.1
 IMAGE_HAPROXY=codenvy/haproxy:1.5.18
-IMAGE_PUPPET=codenvy/puppet
+IMAGE_PUPPET=codenvy/puppet:4.6.2
```

Comment:
it's strange to see version that is not matching repository version
because codenvy/puppet is defined in this repository